examples/fluids/qfunctions/blasius.h

```diff
@@ -13,107 +13,34 @@
-    9.447266795705382414e-01, 9.538467037387058367e-01, 9.617266968332524035e-01, 9.684819213624265011e-01, 9.742272083384174719e-01,
-    9.790747253056680810e-01, 9.831320868743089747e-01, 9.865008381344084754e-01, 9.892753192614093249e-01, 9.915419001656551323e-01,
-    9.968788209317821503e-01, 9.989728724371175206e-01, 9.996990677381791812e-01, 9.999216041491896245e-01, 9.999818594083667023e-01,
-    9.999962745365539307e-01, 9.999993214550036980e-01, 9.999998904550418954e-01, 9.999999843329338001e-01, 9.999999980166356384e-01};
-
-  CeedScalar fpp_table[] = {
-    3.320573362157903663e-01, 3.320379743512646420e-01, 3.319024760665882368e-01, 3.315350015070190337e-01, 3.308206767975666041e-01,
-    3.296466995822193158e-01, 3.279038639411161471e-01, 3.254884713737624113e-01, 3.223045750196085746e-01, 3.182664816607024272e-01,
-    3.133014118810801829e-01, 3.073521951089355775e-01, 3.003798556086043625e-01, 2.923659305537876785e-01, 2.833143548208253981e-01,
-    2.732527514995234941e-01, 2.622329840371728227e-01, 2.503308560706500874e-01, 2.376448876931176457e-01, 2.242941499773744018e-01,
-    2.104151994284793603e-01, 1.961582158440171031e-01, 1.816825052623964043e-01, 1.671515786102889534e-01, 1.527280512426029968e-01,
-    1.385686249977987894e-01, 1.248194106805364800e-01, 1.116118251613979206e-01, 9.905925581301598670e-02, 8.725462988794610575e-02,
-    7.626896310981794158e-02, 6.615089622448211415e-02, 5.692716644118058639e-02, 4.860390768479891377e-02, 4.116863313890323922e-02,
-    3.459272784597366285e-02, 2.883426862493499582e-02, 2.384099224121952881e-02, 1.955324839409207718e-02, 1.590679868531958210e-02,
-    6.578593141419011685e-03, 2.402039843751689954e-03, 7.741093231657678389e-04, 2.201689553063347941e-04, 5.526217815680267893e-05,
-    1.224092624232004387e-05, 2.392841910090350858e-06, 4.127879363882133676e-07, 6.284244603762621373e-08, 8.442944409712819646e-09};
-  // *INDENT-ON*
-
-  CeedScalar nu = newt_ctx->mu / rho;
-  CeedScalar eta = y*sqrt(Uinf/(nu*(x0+x)));
-  CeedInt idx=-1;
-
-  for(CeedInt i=0; i<nprofs; i++) {
-    if (eta < eta_table[i]) {
-      idx = i;
-      break;
-    }
-  }
-  CeedScalar f, fp, fpp;
-
-  if (idx > 0) { // eta within the bounds of eta_table
-    CeedScalar coeff = (eta - eta_table[idx-1]) / (eta_table[idx] - eta_table[idx
-                       -1]);
-
-    f   = f_table[idx-1]   + coeff*( f_table[idx]   - f_table[idx-1] );
-    fp  = fp_table[idx-1]  + coeff*( fp_table[idx]  - fp_table[idx-1] );
-    fpp = fpp_table[idx-1] + coeff*( fpp_table[idx] - fpp_table[idx-1] );
-  } else { // eta outside bounds of eta_table
-    f   = f_table[nprofs-1];
-    fp  = fp_table[nprofs-1];
-    fpp = fpp_table[nprofs-1];
-    eta = eta_table[nprofs-1];
-  }
-
-  *u = Uinf*fp;
-  *t12 = rho*nu*Uinf*fpp*sqrt(Uinf/(nu*(x0+x)));
-  *v = 0.5*sqrt(nu*Uinf/(x0+x))*(eta*fp - f);
+#include "blasius_exact.h"
```

Comment:
I think `ChebyshevEval` could just move here, and the struct can remain so we don't need `blasius_exact.h` at all. (I think it doesn't justify the extra header just for `ChebyshevEval`. This is assuming we move the SNES solver and residual over to `blasius.c`.)